globalization/localization.md

```diff
@@ -294,3 +298,4 @@ Make sure to:
 ## See Also
 
 * [Globalization Overview]({%slug globalization-overview%})
+* [Localize Selected Localization Keys]({%slug common-kb-localize-selected-localization-keys%})
```

Comment:
The suggestion below avoids repetition and provides a simpler alternative wording.

```suggestion
* [Localize Only Some Component Labels]({%slug common-kb-localize-selected-localization-keys%})
```